frontend/src/v5/ui/routes/dashboard/projects/calibration/calibrationStep/verticalSpatialBoundariesHandler/verticalSpatialBoundariesHandler.component.tsx

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
Calibrating an SVG crashes the app after confirming 2d calibration
![Screenshot 2024-07-17 165922](https://github.com/user-attachments/assets/c533add0-532e-4c52-bce0-78561032a2b9)

Reply:
I haven't had this problem recently, so I think my changes might have fixed it